templates/zerver/login.html

```diff
@@ -145,7 +145,7 @@ <h1 class="get-started">{{ _("Log in to Zulip") }}</h1>
 
                             {% if github_auth_enabled %}
                             <div class="login-social">
-                                <form id='social_login_form' class="form-inline github-wrapper" action="{{ url('login-social', args=('github',)) }}" method="get">
+                                <form class="social_login_form form-inline github-wrapper" action="{{ url('login-social', args=('github',)) }}" method="get">
                                     <input type="hidden" name="next" value="{{ next }}">
                                     <button class="login-social-button">
                                         {{ _('Log in with %(identity_provider)s', identity_provider="GitHub") }}
```

Comment:
Are the login and register form blocks identical aside from `google/Google` vs. `github/GitHub` at this point?  If so, we can do a follow-up refactor to use a template loop to emit these so that adding new auth backends no longer requires touching the HTML template files.

(Or potentially, a prep commit refactor to the GitHub/AzureAD ones to be generated that way, and then the Google commit can just remove the block here and configure the code that supports that loop?)

Reply:
Done, though I had a question we didn't have a block for Azure in `accounts_home.html`, we don't support registering by Azure AD?